src/bio.c

```diff
@@ -213,46 +213,47 @@ int wolfSSL_BIO_read(WOLFSSL_BIO* bio, void* buf, int len)
         if (bio->method && bio->method->readCb) {
             ret = bio->method->readCb(bio, (char*)buf, len);
         }
+        else {
```

Comment:
I think that this would be much more readable as a switch-case statement. If this is changed to a switch then `wolfSSL_BIO_read` should follow suit.